spec/suites/dom/DomUtilSpec.js

```diff
@@ -53,6 +53,31 @@ describe('DomUtil', function () {
 		});
 	});
 
+	describe('getClass', function () {
+		it('should return the element\'s class', function () {

```

Comment:
I think it's better to keep the test descriptions consistent and omit the 'should' here.
```suggestion
		it('returns the element\'s class', function () {

```